ocaml/testsuite/tests/typing-layouts-arrays/float_u_array.ml

```diff
@@ -0,0 +1,479 @@
+(**************************************************************************)
```

Comment:
The reason it's not in stdlib is because the closure backend won't build if we add it. I believe there are some plans to move all the jane extensions bits into otherlibs. @dkalinichenko-js knows more about that.

In the short term, it's probably fine to leave this file in tests.

Reply:
Should `gen_u_array.ml` get a similar comment?

Reply:
`gen_u_array.ml` works by wrapping unboxed ints in function closures. That's fine for testing but may not be the best approach for a library. Added related comment.